dataactvalidator/filestreaming/loadFile.py

```diff
@@ -199,9 +201,37 @@ def formatInternalTas(row):
     ])
     return tas
 
+def getSF133List(localSF133Dir):
+    """Return info about existing SF133 files as a list of named tuples."""
+    SF133File = namedtuple('SF133', ['full_file', 'file'])
+    if localSF133Dir is not None:
+        logger.info('Loading local SF-133')
+        # get list of SF 133 files in the specified local directory
+        sf133_files = glob.glob(os.path.join(localSF133Dir, 'sf_133*.csv'))
+        sf133_list = [SF133File(sf133, os.path.basename(sf133)) for sf133 in sf133_files]
+    else:
+        logger.info("Loading SF-133")
+        if CONFIG_BROKER["use_aws"]:
+            # get list of SF 133 files in the config bucket on S3
+            s3connection = boto.s3.connect_to_region(CONFIG_BROKER['aws_region'])
+            s3bucket = s3connection.lookup(CONFIG_BROKER['aws_bucket'])
+            # get bucketlistresultset with all sf_133 files
+            sf133_files = s3bucket.list(
+                prefix='{}/sf_133'.format(CONFIG_BROKER['sf_133_folder']))
+            sf133_list = [SF133File(sf133, os.path.basename(sf133.name)) for sf133 in sf133_files]
+
+    return sf133_list or []
```

Comment:
There's a branch where the `sf133_list` variable isn't defined, but it may not be sensible:
`localSF133Dir is None` and `CONFIG_BROKER["use_aws"] == False`